sdk/ai/Azure.AI.Projects/src/Custom/AIProjectClient.cs

```diff
@@ -37,10 +39,22 @@ public AIProjectClient(string connectionString, TokenCredential credential, AIPr
                   credential,
                   options)
         {
+            _connectionString = connectionString;
         }
 
         private ChatCompletionsClient _chatCompletionsClient;
         private EmbeddingsClient _embeddingsClient;
+        private AgentsClient _agentsClient;
```

Comment:
For now, this is fine. But as @ShivangiReja implements the abstractions, we should remove the dependency from the project client to the agents client, and the agents client should be accessible off the project using the abstraction. Same with embeddings and completions.

Reply:
Currently agents can be used without project dependency. Azure.AI.Project is used just a s a hub for the rest of clients.

Reply:
Yeah, I understand that. And once @ShivangiReja's feature is in, the project client will be able to be a hub for Agents without this field.